packages/rum-core/test/common/apm-server.spec.js

```diff
@@ -179,8 +180,32 @@ describe('ApmServer', function() {
     }, 300)
   })
 
+  it('should log errors from apm-server', done => {
+    const apmServer = new ApmServer(configService, loggingService)
+    apmServer
+      .sendTransactions([
+        {
+          id: '21312',
+          span_count: 0,
+          duration: 100,
+          type: 'app'
+        }
+      ])
+      .catch(err => {
+        /**
+         * APM server error varies by stack, So we check for
+         * explicit characters instead of whole message
+         */
+        expect(err.message).toContain(
+          'validating JSON document against schema: I[#] S[#] doesn\'t validate with "transaction#"'
+        )
+        expect(err.message).toContain('missing properties: "trace_id"')
+      })
+      .then(done)
+  })
+
   it('should report http errors for queued errors', function(done) {
-    spyOn(loggingService, 'warn').and.callThrough()
+    spyOn(loggingService, 'warn')
```

Comment:
Is there any reason to remove the `callThrough`? I think at least it ensures there are no errors thrown in the actual method.

Reply:
You would see the logs everywhere when the test runs. If there is error then you would see them as args in spy so not really different.